Packs/SpringRCEs/Playbooks/playbook-Spring_Core_and_Cloud_Function_SpEL_RCEs.yml

```diff
@@ -1914,7 +1947,76 @@ tasks:
     quietmode: 0
     isoversize: false
     isautoswitchedtoquietmode: false
-system: true
+  "52":
+    id: "52"
+    taskid: ca5742d3-7aec-4c37-829c-d45026b35113
+    type: regular
+    task:
+      id: ca5742d3-7aec-4c37-829c-d45026b35113
+      version: -1
+      name: Search for Vulnerable hosts on Prisma Cloud
+      description: Search configuration inventory on the Prisma Cloud platform using RQL language. Use this command for all queries that start with "config". When no absolute time nor relative time arguments are provided, the default time range is all times.
+      script: '|||prisma-cloud-config-search'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "35"
+    scriptarguments:
+      query:
+        simple: config from cloud.resource where finding.type = 'Host Vulnerability' AND protection.finding.name = 'CVE-2021-44228'
```

Comment:
should be:
config from cloud.resource where finding.type = 'Host Vulnerability' AND protection.finding.name IN ('CVE-2022-22963', 'CVE-2022-22965')